geoapps/inversion/__init__.py

```diff
@@ -9,3 +9,15 @@
 
 from .constants import default_ui_json
 from .params import InversionBaseParams
+
+# from .potential_fields.gravity.driver import GravityDriver
+# from .potential_fields.magnetic_scalar.driver import MagneticScalarDriver
+# from .potential_fields.magnetic_vector.driver import MagneticVectorDriver
+# from .electricals.direct_current.three_dimensions.driver import DirectCurrent3DDriver
+# from .electricals.direct_current.two_dimensions.driver import DirectCurrent2DDriver
+# from .electricals.direct_current.pseudo_three_dimensions.driver import DirectCurrentPseudo3DDriver
+# from .electricals.induced_polarization.three_dimensions.driver import InducedPolarization3DDriver
+# from .electricals.induced_polarization.two_dimensions.driver import InducedPolarization2DDriver
+# # from .electricals.induced_polarization.pseudo_three_dimensions.driver import InducedPolarizationPseudo3DDriver
+# from .natural_sources.magnetotellurics.driver import MagnetotelluricsDriver
+# from .natural_sources.tipper.driver import TipperDriver
```

Comment:
Let's keep those here and create the DRIVER_MAP here if possible (see comment below). Otherwise, clean up...

Reply:
done